js/copy-service/Parser/Parser.ts

```diff
@@ -425,7 +406,7 @@ class Parser {
       }
     }
 
-    const token = _.first(tokens);
+    const token = _.first(tokens)!;
```

Comment:
```suggestion
    const token = tokens[0];
```
I am not sure the point of lodash there